### PR TITLE
new release 0.12.19

### PR DESCRIPTION
**Pull request by arthurzam**
@thesamesam

### CODE REVIEW

File: NEWS.rst [hunk 1]
```diff
@@ -2,6 +2,23 @@
 Release Notes
 =============
 
+----------------------------
+pkgcore 0.12.19 (2023-01-19)
+----------------------------
+
+- properly close open file handlers during tests (Arthur Zamarin)
```
[the remark this answers is not in the document]
Comment (arthurzam) [reply NEWS.rst]: No problem on changing the style - I'm trying to create a nice view for release notes, so I'll take any improvement.

Adding subsections might be bad, when viewed in sphinx news

[the remark this answers is not in the document]
Comment (thesamesam) [reply NEWS.rst]: Let's do generic first (so items like "properly close", "add better docs for the foo" first as different items, like now), then specific items sorted by name after.

File: NEWS.rst [hunk 2]
```diff
@@ -2,6 +2,24 @@
...
+pkgcore 0.12.19 (2023-01-19)
+----------------------------
+
+- domain: fix parsing of multiple USE_EXAPNDs
+  (Arthur Zamarin, #391, https://bugs.gentoo.org/891409)
+
+- portage_conf: add support for make.profile as its own directory, behaving as
+  specialized user profile (Arthur Zamarin, #390)
+
+General:
```
Comment (thesamesam) [inline NEWS.rst:15]: I like this, but it's ok with me if we drop the general and just have the layout match this wrt sorting.

File: NEWS.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (arthurzam) [reply NEWS.rst]: > Let's do generic first (so items like "properly close", "add better docs for the foo" first as different items, like now), then specific items sorted by name after.

Sorry, but after multiple tries, I'm unable to understand this.
I was thinking maybe sort it with "new features", "bugs fixed", "random"

[the remark this answers is not in the document]
Comment (thesamesam) [reply NEWS.rst]: Now:
```
- domain: fix parsing of multiple USE_EXAPNDs
  (Arthur Zamarin, #391, https://bugs.gentoo.org/891409)

- portage_conf: add support for make.profile as its own directory, behaving as
  specialized user profile (Arthur Zamarin, #390)

- re-add Brian Harring as author for pkgcore

- properly close open file handlers during tests (Arthur Zamarin)

- various code cleanups, refactoring and modernization (Brian Harring)
```

My suggestion, maybe, so the specificity narrows as you read downwards:
```
- re-add Brian Harring as author for pkgcore

- properly close open file handlers during tests (Arthur Zamarin)

- various code cleanups, refactoring and modernization (Brian Harring)

- domain: fix parsing of multiple USE_EXAPNDs
  (Arthur Zamarin, #391, https://bugs.gentoo.org/891409)

- portage_conf: add support for make.profile as its own directory, behaving as
  specialized user profile (Arthur Zamarin, #390)
```

And maybe (I don't know if I like this):
```
- general: re-add Brian Harring as author for pkgcore

- general: properly close open file handlers during tests (Arthur Zamarin)

- general: various code cleanups, refactoring and modernization (Brian Harring)

- domain: fix parsing of multiple USE_EXAPNDs
  (Arthur Zamarin, #391, https://bugs.gentoo.org/891409)

- portage_conf: add support for make.profile as its own directory, behaving as
  specialized user profile (Arthur Zamarin, #390)
```

[the remark this answers is not in the document]
Comment (arthurzam) [reply NEWS.rst]: I see and understand it, let's try it

**Review by thesamesam — APPROVED** [review]

**Pull request merged**